packages/gatsby/cache-dir/head/index.js

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import { Helmet } from "react-helmet-async"
+import { useLocation } from "@reach/router"
+
+export const Head = props => {
+  const location = useLocation()
+
+  // This supports passing a gatsby-plugin-image object to the image prop, e.g.
+  // image={getImage(data.metaImage)}
+  const image = props.image?.images?.fallback?.src || props.image
+  const { title, description } = props
+
+  return (
+    <Helmet>
+      {/* General meta tags */}
+      <meta property="og:type" content="website" />
+      <meta property="og:url" content={location.href} />
+
+      {title && (
+        <>
+          <title>{title}</title>
+          <meta name="twitter:title" content={title} />
+          <meta property="og:title" content={title} />
+        </>
+      )}
+
+      {description && (
+        <>
+          <meta name="description" content={description} />
+          <meta name="twitter:description" content={description} />
+          <meta property="og:description" content={description} />
+        </>
+      )}
+
+      {image && (
```

Comment:
I'm missing `og:image:alt` and `twitter:image:alt` here. Could be the the description.

Reply:
It don't think setting this to the description is sensible (we already send that data to the social media services anyway, why duplicate it?), but we _could_ expose an `imageAlt` prop for users to set it. 

However, it's not clear if even setting this at all is a good idea (ref https://github.com/Yoast/wordpress-seo/issues/12210):

> Specifically, alt attributes are typically used to describe the content of an image, rather than the role of an image. In the context of a linked image in a social share, this is incorrect. A linked image should describe the expected behaviour of the link, not the visual content of an image.
> Addressing this requires, theoretically, that an image have different alt attributes for differing contexts. We believe that this problem should be addressed by/at/in Facebook, rather than by forcing content managers to micromanage their media by context.

I'd say we leave this as-is for now and potentially consider adding this later!

Reply:
Heh, seems like a can of worms be probably shouldn’t open. I've been told in the past that it’s valuable but didn’t know about that notion. Thanks!